models/engine/db_storage.py

```diff
@@ -30,14 +30,19 @@ class DBStorage:
 
     def __init__(self):
         """ creates connection to db"""
-        user = os.getenv('HBNB_MYSQL_USER')
-        passwd = os.getenv('HBNB_MYSQL_PWD')
-        host = os.getenv('HBNB_MYSQL_HOST')
-        database = os.getenv('HBNB_MYSQL_DB')
+        user = getenv('HBNB_MYSQL_USER')
+        passwd = getenv('HBNB_MYSQL_PWD')
+        host = getenv('HBNB_MYSQL_HOST')
+        database = getenv('HBNB_MYSQL_DB')
         self.__engine = create_engine('mysql+mysqldb://{}:{}@{}/{}'
                                       .format(user, passwd, host, database))
-        if os.getenv('HBNB_ENV') == 'test':
-            Base.metadata.drop_all(self.__engine)
+        if getenv('HBNB_ENV') == 'test':
```

Comment:
Graceful way to handle an error. It's not clear to me if the user decides to not drop the tables, where will it be handled?

Reply:
It will end execution. I don't want to drop all of my dev db